klient/k8s/resources/resources.go

```diff
@@ -184,16 +189,19 @@ func (r *Resources) GetScheme() *runtime.Scheme {
 	return r.scheme
 }
 
-func (r *Resources) Watch(ctx context.Context, object cr.ObjectList, opts *cr.ListOptions, cfg *rest.Config) *watcher.EventHandlerFuncs {
-	cl, err := cr.NewWithWatch(cfg, cr.Options{})
-	if err != nil {
-		return nil
-	}
+func (r *Resources) Watch(ctx context.Context, object k8s.ObjectList, opts ...ListOption) *watcher.EventHandlerFuncs {
```

Comment:
@ShwethaKumbla Let's remove ctx from the signature here and used it when needed in `Start(ctx)`